app/services/iaa_reporting_helper.rb

```diff
@@ -26,12 +26,15 @@ def key
   )
 
   # @return [Array<IaaConfig>]
-  def iaas
+  def iaas(**options)
+    filtered_issuers = options[:filtered_issuers]
```

Comment:
we can use optional keywords here:
```suggestion
  def iaas(filtered_issuers: nil)
```

Reply:
oh wait this option looks completely unused?

Reply:
This is a symptom of an older solution, will remove